Changes

```diff
@@ -1,8 +1,6 @@
 Change history for libwww-perl
 
 {{$NEXT}}
-
-6.39      2019-03-27 19:00:00Z
```

Comment:
👍

Reply:
I checked a previous pull request and made changes accordingly. Now waiting for Travis test (but they are experiencing issues and the services is downgraded)

Reply:
Good stuff.  Also, you don't need to create a new pull request even with substantial changes.  `git push --force` will overwrite your previous commits and the pull request will reflect that, which allows us to continue discussion in the same place.

Reply:
Good to know for next time - it seems it passes the test now. BTW I am not doing this via git, but via the site (don't ask ;-)) - too late I saw that I could apply changes to the comparing branch and have the pull request updated. We're getting there...